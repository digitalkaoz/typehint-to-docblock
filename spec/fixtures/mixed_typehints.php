<?php

namespace Foo;

use Bar\Foo;
use Lol\Cat;

class Test
{
    public function it_does_things(Foo $foo, array $lol, Cat $cat)
    {
    }
}
