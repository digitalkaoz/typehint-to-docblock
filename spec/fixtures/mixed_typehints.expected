<?php

namespace Foo;

use Bar\Foo;
use Lol\Cat;
class Test
{
    /**
     * does_things
     *
     * @param \Bar\Foo $foo
     * @param array $lol
     * @param \Lol\Cat $cat
     */
    public function does_things($foo, array $lol, $cat)
    {
    }
}